### DISCUSSION TITLE
Add BTCPayServer to DigitalOcean's 1-click marketplace

### DISCUSSION THREAD

**Discussion by heredia21** [btcpayserver/btcpayserver#5491] [Ideas / Feature Requests]
In an effort to increase adoption for bitcoin payments and BTCPayServer I think it would be helpful if someone with technical knowledge can become a Vendor at DigitalOcean's marketplace to add/manage BTCPay Server:

Marketplace: https://marketplace.digitalocean.com/
Become a vendor: https://marketplace.digitalocean.com/vendors

**Comment by dennisreimann** [comment]
Good idea. Ping @pavlenex

**Comment by dstrukt** [comment]
cACK as well.

**Comment by heredia21** [comment]
Any updates?
